hail/python/hailtop/utils/utils.py

```diff
@@ -116,7 +116,7 @@ def grouped(n: int, ls: List[Any]) -> Iterable[Any]:
         yield group
 
 
-def partition(k: int, ls: List[int]) -> Iterable[Any]:
+def partition(k: int, ls: List[int]) -> Iterable[List]:
```

Comment:
What is the type of the elements of the List? Take a look at the type of `ls` for a hint.